pkg/listener/send_result.go

```diff
@@ -28,20 +27,23 @@ func NewSendResultListener(targets *target.Collection) report.PolicyReportResult
 					result.Resources = []corev1.ObjectReference{*re.GetScope()}
 				}
 
+				resultsToSend := []v1alpha2.PolicyReportResult{}
```

Comment:
this listener should only send the result its receiving, why not revert to the original state to recreate the old behavior?